package.nls.json

```diff
@@ -435,5 +435,6 @@
     "comment": [
       "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered."
     ]
-  }
+  },
+  "generateOptionsSchema.useVSDbg.description": "Enable new .NET 8+ Mono Debugger (preview)"
```

Comment:
>Mono [](http://example.com/codeflow?start=68&length=4)

In VS Code, will this option apply to anything besides WASM? (since the MAUI scenarios are already on vsdbg right? Or is that still optional?)

Reply:
For MAUI the default is enabled and the option is in their extension. I would like to unify it and use only this option in the future. But it's okay, maybe this is very complex, I will change mono to wasm here.